template.Dockerfile

```diff
@@ -13,35 +13,18 @@ ENV PYTHONFAULTHANDLER=1 \
   VIRTUALENV_PIP=21.2.1 \
   MYPY_VERSION=0.930
 
-
 RUN pip install "poetry==$POETRY_VERSION" "mypy==$MYPY_VERSION"
-# pylintrc, coveragerc, poetry.lock and pyproject.toml shall not change very
-# often, so it is a good idea to add them as soon as possible
-COPY .coveragerc mypy.ini .flake8  ./
-COPY poetry.lock pyproject.toml ./
+COPY .coveragerc mypy.ini .flake8 poetry.lock pyproject.toml ./
+
 # During make build this sed command is substituted by 's/secc/evcc/g'
 RUN sed -i 's/secc/secc/g' pyproject.toml
 
-# Due to an issue with Python 3.10 and poetry, if we use a poetry virtual env,
-# we need to disable the option: poetry config experimental.new-installer false
-# check https://github.com/python-poetry/poetry/issues/4210
-# However, if we run poetry config virtualenvs.create false, then we dont.
-# Do not create a virtual poetry env as we already are in an isolated container
-RUN poetry config virtualenvs.create false
 # Install dependencies and the project in the venv
 RUN poetry install --no-interaction --no-ansi
 
 # Copy the project to the system
 COPY iso15118/ iso15118/
 
-# Run the tests and linting
-COPY tests/ tests/
-RUN poetry run black --check --diff --line-length=88 iso15118 tests
-RUN poetry run flake8 --config .flake8 iso15118 tests
-# RUN poetry run mypy --config-file mypy.ini iso15118 tests
-RUN poetry run pytest -vv --cov-config .coveragerc --cov-report term-missing  --durations=3 --cov=.
```

Comment:
why did you remove this?

Reply:
I'm not sure it's good practice to run tests as part of a docker image build, the flow mentioned here is more standard: https://stackoverflow.com/questions/67088512/should-i-run-tests-during-the-docker-build

Reply:
ok, I see the point, but if we remove it from the dockerfile then we need to provide an alternative to run the tests, otherwise there wont be running tests

Reply:
Done in #147